ceph.go

```diff
@@ -270,15 +272,16 @@ func (r mapping) String() string {
 	return fmt.Sprintf("%d->%d", r.From, r.To)
 }
 
-func mustGetOsdsForBucket(bucket string) []int {
-	osds, err := getOsdsForBucket(bucket)
+func mustGetOsdsForBucket(bucket string, device_class string) []int {
```

Comment:
For consistency, `device_class` should be camelCase (`deviceClass`). (Here and elsewhere.)